azure/values-azure-secondary.yaml

```diff
@@ -9,26 +10,28 @@ kubecostModel:
   warmCache: false
   warmSavingsCache: false
   etl: false
+  etlCloudAsset: false
   maxQueryConcurrency: 1
 global:
   grafana:
     enabled: false
     proxy: false
-  alertmanager:
-    enabled: false
 prometheus:
   server:
     global:
       external_labels:
+        # cluster_id should be unique for all clusters and the same value as .kubecostProductConfigs.clusterName
         cluster_id: kubecostProductConfigs_clusterName
     retention: 2d
+  alertmanager:
+    enabled: false
```

Comment:
I think we can safely just remove the alert manager block as it's disabled by default.